mlflow/langchain/utils/__init__.py

```diff
@@ -104,71 +100,39 @@ def picklable_runnable_types():
     """
     from langchain.chat_models.base import SimpleChatModel
     from langchain.prompts import ChatPromptTemplate
+    from langchain.schema.runnable import RunnableLambda, RunnablePassthrough
 
-    types = (
+    return (
         SimpleChatModel,
         ChatPromptTemplate,
+        RunnablePassthrough,
+        RunnableLambda,
     )
 
-    try:
-        from langchain.schema.runnable import (
-            RunnableLambda,
-            RunnablePassthrough,
-        )
-
-        types += (RunnableLambda, RunnablePassthrough)
-    except ImportError:
-        pass
-
-    return types
-
 
 @lru_cache
 def lc_runnable_with_steps_types():
-    # import them separately because they are added
-    # in different versions of langchain
-    try:
-        from langchain.schema.runnable import RunnableSequence
+    from langchain.schema.runnable import RunnableParallel, RunnableSequence
 
-        types = (RunnableSequence,)
-    except ImportError:
-        types = ()
-
-    try:
-        from langchain.schema.runnable import RunnableParallel
-
-        types += (RunnableParallel,)
-    except ImportError:
-        pass
-
-    return types
+    return (RunnableParallel, RunnableSequence)
 
 
 def lc_runnable_assign_types():
-    try:
-        from langchain.schema.runnable.passthrough import RunnableAssign
+    from langchain.schema.runnable.passthrough import RunnableAssign
 
-        return (RunnableAssign,)
-    except ImportError:
-        return ()
+    return (RunnableAssign,)
 
 
 def lc_runnable_branch_types():
-    try:
-        from langchain.schema.runnable import RunnableBranch
+    from langchain.schema.runnable import RunnableBranch
 
-        return (RunnableBranch,)
-    except ImportError:
-        return ()
+    return (RunnableBranch,)
 
 
 def lc_runnable_binding_types():
```

Comment:
Not a blocker, but do we still need these functions?

Reply:
We could convert to constant, but it needs a bit of refactoring because this file then imports langchain globally. Lemme keep them for now.